### PR TITLE
Jammy hotplug

### PR DESCRIPTION
**Pull request by aciba90**
# rebase

Hotplug changes for the next release.

https://warthogs.atlassian.net/browse/SC-1718

### CODE REVIEW

**Review by TheRealFalcon — APPROVED** [review]

**Comment by blackboxsw** [comment]
In the interest of getting some integration test runs on this overnight. I've merge this approved PR.

File: debian/cloud-init.postinst [hunk 1]
```diff
@@ -371,6 +371,11 @@ change_cloud_init_output_log_permissions() {
     fi
 }
 
+rename_hook_hotplug_udev_rule() {
+    # Avoids LP: #1946003 see commit: b519d861aff8b44a0610c176cb34adcbe28df144
+    mv -f /etc/udev/rules.d/10-cloud-init-hook-hotplug.rules \
+        /etc/udev/rules.d/90-cloud-init-hook-hotplug.rules
```
Comment (blackboxsw) [inline debian/cloud-init.postinst:377]:
```suggestion
   if [ -f /etc/udev/rules.d/10-cloud-init-hook-hotplug.rules ]; then
        mv -f /etc/udev/rules.d/10-cloud-init-hook-hotplug.rules \
            /etc/udev/rules.d/90-cloud-init-hook-hotplug.rules
   fi
```

**Review by blackboxsw — CHANGES_REQUESTED** [review]

File: debian/cloud-init.postinst (hunk 1 above)
Comment (blackboxsw) [reply debian/cloud-init.postinst]: Squash merged this changeset since we hadn't landed yet.

**Review by blackboxsw — APPROVED** [review]
+1 unrelated docs jobs failing at the moment through CI

**Pull request merged**